src/Http/Controllers/Api/Geojson/ImageAnnotationsController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace Biigle\Modules\Geo\Http\Controllers\Api\Geojson;
+
+use DB;
+use Biigle\Image;
+use GeoJson\Feature\{Feature, FeatureCollection};
+use GeoJson\Geometry\Point;
+use Biigle\Http\Controllers\Api\Controller;
+use League\Flysystem\FileNotFoundException;
+
+class ImageAnnotationsController extends Controller{
+
+  public function index($id)
+  {
+    $image = Image::findOrFail($id);
+    $this->authorize('access', $image);
+
+    $metadata = $image->metadata;
+    $image_width_m = 2 * (float)$image->metadata['distance_to_ground'];
+    $result = $image->annotations->map(function($annotation) use($image, $metadata,$image_width_m){
```

Comment:
Please use more spaces as shown below everywhere:
```suggestion
    $result = $image->annotations->map(function ($annotation) use ($image, $metadata, $image_width_m) {
```